lib/model/query/entities.js

```diff
@@ -849,7 +873,95 @@ CROSS JOIN
 const del = (entity) => ({ run }) =>
   run(markDeleted(entity));
 
-del.audit = (entity, dataset) => (log) => log('entity.delete', entity.with({ acteeId: dataset.acteeId }), { uuid: entity.uuid });
+del.audit = (entity, dataset) => (log) =>
+  log(
+    'entity.delete',
+    entity.with({ acteeId: dataset.acteeId }),
+    { entity: { uuid: entity.uuid } }
+  );
+
+////////////////////////////////////////////////////////////////////////////////
+// RESTORE ENTITY
+
+const restore = (entity) => ({ run }) =>
+  run(markUndeleted(entity));
+
+restore.audit = (entity, dataset) => (log) =>
+  log(
+    'entity.restore',
+    entity.with({ acteeId: dataset.acteeId }),
+    { entity: { uuid: entity.uuid } }
+  );
+
+////////////////////////////////////////////////////////////////////////////////
+// PURGE DELETED ENTITIES
+
+const PURGE_DAY_RANGE = config.has('default.taskSchedule.purge')
+  ? config.get('default.taskSchedule.purge')
+  : 30; // Default is 30 days
```

Comment:
Maybe we should move this to central place, this is present in three file at the moment.

Reply:
Agreed, maybe it could be injected in the container around [here](https://github.com/getodk/central-backend/blob/master/lib/model/container.js#L79) 

```
  // Inject some constants
  const PURGE_DAY_RANGE = config.has('default.taskSchedule.purge')
    ? config.get('default.taskSchedule.purge')
    : 30; // Default is 30 days
  container.constants = { PURGE_DAY_RANGE };
```

Reply:
The separate constant file looks good to me!